### ISSUE TITLE
[feat] different `shellcheck.exclude` for different file

### ISSUE THREAD

**Issue by loynoir** [vscode-shellcheck/vscode-shellcheck#998]
### Current
`shellcheck.exclude: shellcheckRuleString[]`

### Feat
`shellcheck.exclude: shellcheckRuleString[] | Record<GlobPatternString, shellcheckRuleString[]>`

**Comment by felipecrs** [comment]
I agree with @timonwong, and thus I'm closing this issue. But feel free to continue the discussion here if you want.
